src/exchange/swap/getExchangeRates.js

```diff
@@ -84,26 +84,29 @@ const getExchangeRates: GetExchangeRates = async (
       } else {
         // NB Allows us to simply multiply satoshi values from/to
         magnitudeAwareRate = (tradeMethod === "fixed"
-          ? BigNumber(rate)
+          ? BigNumber(maybeRate)
```

Comment:
Since it's a maybe, shouldn't we fallback to 0 ?

Reply:
I couldn't find a better name, if it's "fixed", as used in this ternary it won't be a maybe since all fixed rates have a `rate`, line 100 fallback to the calculation so it's never undefined when accessed if that makes sense

Reply:
Yeah but isn't it problematic to have magnitudeAwareRate (which is returned, i don't know how it's used after) being a NaN bignumber ( BigNumber(undefined) ) ?

Reply:
`magnitudeAwareRate` will never be `BigNumber(undefined)`. If the `tradeMethod` is fixed, the rate exists.

Reply:
mmh ok, did not know the specifics of this part of the code.
LGTM then